### PR TITLE
Declare request wrapper keys as constants

### PR DESCRIPTION
**Pull request by guscarreon**
In pull request #2378 we decided to put request wrapper keys into constants. This is the second part of said endevour, coverts `"consent"`, `"amp"`, `"eids"`, `"gdpr"`, `"prebid"`, `"schain"` and `"us_privacy"`.

### CODE REVIEW

**Comment by SyntaxNode** [comment]
Why? Does this provide a performance gain of some kind?

**Comment by guscarreon** [comment]
> Why? Does this provide a performance gain of some kind?

None that I can think of. We thought the constants are easier to manage.

**Review by VeronikaSolovei9 — DISMISSED** [review]
Looks like a good change

File: openrtb_ext/request_wrapper.go [hunk 1]
```diff
@@ -44,8 +44,15 @@ type RequestWrapper struct {
 
 const (
 	jsonEmptyObjectLength               = 2
-	ConsentedProvidersSettingsStringKey = "ConsentedProvidersSettings"
-	ConsentedProvidersSettingsListKey   = "consented_providers_settings"
+	consentedProvidersSettingsStringKey = "ConsentedProvidersSettings"
+	consentedProvidersSettingsListKey   = "consented_providers_settings"
+	consent                             = "consent"
```
Comment (mansinahar) [inline openrtb_ext/request_wrapper.go:49]: @guscarreon I like the change from hardcoded strings to constants. Thanks for doing that. However, one minor comment, for these constant names we should decide on whether we want the `Key` suffix for them or not and keep it consistent among all constant names that represent a hardcoded string. I personally lean towards having the suffix but don't feel too strongly about it so will leave it up to you.

**Review by VeronikaSolovei9 — APPROVED** [review]

**Pull request merged**